source/agora/node/Registry.d

```diff
@@ -273,21 +282,48 @@ public class NameRegistry: NameRegistryAPI
 
         auto ptr = public_key in registry_map;
         // We are authoritative, so we can set `NameError`
-        if (!ptr)
+        if (!ptr || !(*ptr).payload.data.addresses.length)
```

Comment:
I don't think that is possible, payloads with no addresses are rejected anyways.